beacon-chain/core/time/slot_epoch.go

```diff
@@ -60,6 +60,16 @@ func CanUpgradeToAltair(slot types.Slot) bool {
 	return epochStart && altairEpoch
 }
 
+// CanUpgradeToMerge returns true if the input `slot` can upgrade to Merge fork.
+//
+// Spec code:
```

Comment:
I wonder if we should start sticking the Spec code inside the function or something, otherwise I assume this is going to show up in the godoc and I'm not sure if we want it to?

Reply:
I think the day 0 intention is to show them in godoc because they are easier to consume being python for readers to cross reference